packages/es-components/src/components/controls/textbox/Textbox.js

```diff
@@ -24,13 +24,19 @@ const TextboxBase = styled(InputBase)`
 `;
 
 const Textbox = React.forwardRef(function Textbox(props, ref) {
-  const { prependIconName, appendIconName, ...additionalTextProps } = props;
+  const {
+    prependIconName,
+    appendIconName,
+    type,
+    ...additionalTextProps
+  } = props;
   const theme = useTheme();
   const validationState = React.useContext(ValidationContext);
 
   const inputRef = React.useRef();
   React.useImperativeHandle(ref, () => inputRef.current);
 
+  const finalType = type || 'text';
```

Comment:
Do we need `finalType` at all since the default is already set to "text"?

Reply:
I just did it for safety's sake in case someone decided `type={null}` was a good idea at some point.

Reply:
I'd let them have their null in that case. We haven't done double-checks like that in other cases and I think I'd prefer consistency and simplicity of just passing it through.

Reply:
Ok! I'll make the change. 😄

Reply:
Done!

Reply:
Thanks Steve!